include/crow/http_server.h

```diff
@@ -59,9 +59,6 @@ namespace crow
 
         void run()
         {
-            if (concurrency_ < 0)
-                concurrency_ = 1;
```

Comment:
just wondering if you removed this because it never got executed or another reason?

Reply:
Type of this variable is uint16, so it is impossible to fall below 0. Maybe a good idea would be to check if it doesn't equal 0.

Reply:
I've pushed change with checking this variable in constructor.